app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -114,7 +106,17 @@ class Dashboard extends React.Component {
 
   render() {
     if (this.state.display_course_summary) {
-      return <Bar data={this.state.data} />;
+      if (this.state.data.datasets === []) {
+        return (
+          <div>
+            {/*DOESN'T SHOW UP*/}
+            <p>{I18n.t('Create a Marking Scheme to display course summary graph.')}</p>
```

Comment:
I'm not quite sure why this doesn't show up and I'm currently doing research on this

Reply:
@emilychangg In Javascript, `===` compares objects and arrays by identity (i.e., memory address), like Python's `is`. If you want to check if an array is empty, you can check its `.length` attribute and see whether it's 0.